physical_properties/pKa/experimental_data/README.md

```diff
@@ -8,7 +8,11 @@ Since pKa values measured by Sirius T3 were reported with 2 decimal places, we h
 
 ## Manifest
 - `pKa_experimental_values.csv` - Mean and SEM of pKa values organized in a table. Up to three pKa values were reported based on how many pKas were measured in the experimental interval of 2-12.
-- `calc_pKa_value_statistics.py` - Python script used to calculate mean and SEM of pKa values.
-- `pKa_results_of_replicate_experiments.csv` - Experimental results of independent replicates of pKa measurements(N=3) and names of experimental reports (PDF files can be found in `experiment_reports\` directory).
+- `calc_pKa_value_statistics.py` - Python script used to calculate mean and SEM of pKa values from `pKa_results_of_replicate_experiments.csv`.
+- `pKa_results_of_replicate_experiments.csv` - Experimental results of independent replicates of pKa measurements(N=3) and names of experimental reports (PDF files can be found in `experiment_reports\` directory). For molecules with pKas measured both in 100% water and with cosolvent, pKa measurements done in 100% water are selected to be used for SAMPL6 challenge analysis considering potentially higher accuracy of pKa measurements in 100% water.
 - `experiment_reports/` - This directory contains reports generated by Sirius T3 software for the results of pKa measurements. Refer to `pKa_replicate_experimental_results.csv` for matching SAMPL6 molecule IDs to names of report files.
+- `experiment_reports_of_extra_cosolvent_experiments/` -  This directory contains reports generated by Sirius T3 software for the results of extra pKa measurements performed with cosolvent method (Yasuda-Shedlovsky Extrapolation) for molecules that UV-metric pKa measurement in 100% water  were already done. These extra experiments were collected only to evaluate if there is a bias in pKa measurement method that uses cosolvent.

```

Comment:
> This directory contains reports generated by Sirius T3 software for the results of extra pKa measurements performed with cosolvent method (Yasuda-Shedlovsky Extrapolation) for molecules that UV-metric pKa measurement in 100% water  were already done.

This is a little long/cumbersome, maybe two sentences instead:

This directory contains reports generated by Sirius T3 software for the results of extra pKa measurements performed with cosolvent method (Yasuda-Shedlovsky Extrapolation). These molecules already have UV-metric pKa measurements in 100% water.

Reply:
Done.